logical-operators.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Created by daniel.potts on 20/08/15.
+ */
+//Declare two variables and output the largest
+
+var smallNumber = 10;
+var bigNumber = 13;
+if (bigNumber > smallNumber){
+    console.log(bigNumber);}
+else {
+    console.log(smallNumber)
+}
+
+//Declare three variables and output the product and whether product is negative or not.
+var negcheck = function(value1,value2,value3) {
+    var answer = (value1 * value2) * value3;
```

Comment:
No need for the brackets - order is unimportant!:
a * b * c = (a*b) * c = a* (b *c)